cores/esp8266/FS.h

```diff
@@ -99,12 +119,22 @@ struct FSInfo {
     size_t maxPathLength;
 };
 
+class FSConfig
+{
+public:
+    FSConfig(bool autoFormat = true) {
+        _autoFormat = autoFormat;
+    }
+
```

Comment:
Thanks !
Did you consider using an enum for the constructor parameter, to make it more explicit/visual for the end user (like `FS_AUTOFORMAT` / `FS_NOFORMAT` instead of a single `true` or `false`) ?

Reply:
How about an set/get function instead of an enum?  `setAutoFormat(true);`?

Also, I need to add a poor man's RTTI by putting a type in the shared header so you can't accidentally pass in the wrong config type (well, you could, but the library would catch it).

And after looking at what I had to do w/the testing I think I need a separate `setConfig()` method in FS to make things like "format()" sensible.  Begin will go back to a no-parameter method using a default config if nobody calls setConfig().